version/version.go

```diff
@@ -14,8 +14,15 @@ var (
 
 	// DefaultRootlessImage holds the rootless default image
 	DefaultRootlessImage = "docker.io/moby/buildkit:buildx-stable-1-rootless"
+
+	// DefaultHelperImage holds the image of the helper
+	DefaultHelperImage = ""
```

Comment:
This should be set to the image that gets published by the CI system here before merging.

Reply:
Done.